tests/lib/eslint-tester.js

```diff
@@ -329,6 +329,27 @@ describe("ESLintTester", function() {
         });
     });
 
+    it("should pass-through the parser to the rule", function() {
```

Comment:
It's probably best to include one test with a custom parser (maybe just Esprima) to make sure you can get something other than error message.